docs/apis-tools/frontend-development/01-introduction-to-task-applications.md

```diff
@@ -0,0 +1,130 @@
+
+Task applications play a key role in the orchestration of business processes. They enable the orchestration of processes that still contain manual work instead of automating each process step in advance. This unlocks the potential for continuous improvement and for identifying opportunities for process optimization as well as automation.
+
+:::tip
+Not sure if you should use the Camunda Tasklist, build your custom task application, or use a third-party application? Read the [guide to understand human task management](/components/best-practices/architecture/understanding-human-tasks-management.md#deciding-about-your-task-list-frontend) first.
+:::
+
+## Tasklist layout
+
+Camunda 8 comes with a ready-to-use Tasklist UI that implements all key concepts of a task application. The Tasklist UI is a generic task application; your custom task application should probably be tailored to your specific use case and also include external data sources and tools.
+
+The Tasklist UI is split into two main pages: the [tasks page](#task-page) and the [processes page](#processes-page).
+
+### Task page
+
+The task page lists all tasks that are pending for a user or user group, and allows users to pick and claim a task from that queue to work on. On the same page, the details of a selected task are displayed including the form that the user must submit in order to execute and complete the task.
+
+The task page is divided into two panels, similar to the layout of messaging clients:
+
+<img src={TasklistDetailsImg} className={styles.noShadow} alt="Task details page in Tasklist" />
+
+#### Task queue side panel
+
+The **task queue side panel** lists all tasks that are pending for a user or user group. It comes with filter and sort options that allow users to identify the right task to work on next. The tasks can be sorted by the creation date, due date, or follow-up date.
+
+Learn more how to work with the task queue in the [Tasklist user guide](/components/tasklist/userguide/using-tasklist.md).
+
+#### Task details panel
+
+The **task details panel** shows the details of the selected task. A [form](/guides/utilizing-forms.md) is displayed in the center of the details panel, which must be filled out to complete a task.
+
+:::tip
+Typically, a task application utilizes forms to capture information from the user, to make a decision, to collect the results from a real-world task, or to provide task instructions to the user.
+
+However, a [user task](/components/modeler/bpmn/user-tasks/user-tasks.md/#user-task-forms) is not limited to forms. A user task could also represent navigating to an external desktop or web application, where a task is to be performed, such as updating a record in a CRM. You can even use them to track physical work or actions using sensors, IoT devices, or any interface that can talk to the web, by registering a [job worker](/components/concepts/job-workers.md) to the job type `io.camunda.zeebe:userTask`.
```

Comment:
Do we want to mention tasks without forms too? (variable view)

Reply:
Ideally, we get rid of the variable view one day (sooner than later), as it imposes data access risks